pyderman/drivers/chrome.py

```diff
@@ -26,8 +26,8 @@ def get_url(
     if not resolved_version:
         raise ValueError(f"Unable to locate ChromeDriver version! [{version}]")
     if _os == "mac-m1":
-        _os = "mac"  # chromedriver_mac64_m1
-        _os_bit = "%s_m1" % _os_bit
+        _os = "mac"  # chromedriver_mac_arm64
+        _os_bit = "_arm%s" % _os_bit
```

Comment:
```suggestion
        _os = "mac"
        if resolved_version > "106.0.5249.21":
            _os_bit = "_arm%s" % _os_bit  # chromedriver_mac_arm64
        else:
            _os_bit = "%s_m1" % _os_bit  # chromedriver_mac64_m1
```

Reply:
Thank you @mentalisttraceur !

You are absolutely right that my change would break downloads for version 88-106, and after testing your suggestion I see it works for older and newer versions, where 106.0.5249.21 is the latest version which still uses the old naming scheme.

This is my first time contributing to someone elses Repository, what would be the appropriate next step to use your suggestion?

Reply:
@KurtLeimeister Happy to help!

If you see a "commit suggestion" button (as shown in step three of [this GitHub doc on applying suggested changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes)), then that's probably the easiest option - it will add my suggestion as a new commit to your PR branch.